app/scripts/views/sign_up.js

```diff
@@ -219,13 +258,24 @@ define(function (require, exports, module) {
       return p()
         .then(function () {
           if (! self._isUserOldEnough()) {
```

Comment:
I think this can be made more clear, something like 

```js
if (! self._coppa.hasValue() || ! self._isUserOldEnough()) {
```

It's explicit that we are attempting to sign in users who have not filled in the coppa field or who aren't old enough.